docs/testplan/transceiver_onboarding_test_plan.md

```diff
@@ -116,23 +118,37 @@ These tests do not require traffic and are standalone, designed to run on a Devi
         }
       },
       "dut_name_1": {
-        "port_1": {
+        "Ethernet0": {
+          "vendor_name": "ACME Corp.",
+          "vendor_pn": "QSFP-100G-AOC-15M",
+          "vendor_sn": "serial_number_001",
+          "vendor_date": "vendor_date_code",
+          "vendor_oui": "vendor_oui",
+          "vendor_rev": "revision_number",
+          "hardware_rev": "hardware_revision_number"
+        },
+        "Ethernet4-6": {
           "vendor_name": "ACME Corp.",
           "vendor_pn": "QSFP-100G-AOC-15M",
           "vendor_sn": "serial_number",
           "vendor_date": "vendor_date_code",
           "vendor_oui": "vendor_oui",
-          "vendor_rev": "revision_number"
+          "vendor_rev": "revision_number",
+          "hardware_rev": "hardware_revision_number"
         },
-        "port_2": {
+        "Ethernet16,Ethernet20,Ethernet24": {
           "vendor_name": "Example & Co",
-          "vendor_pn": "SFP-1000BASE-LX"
+          "vendor_pn": "SFP-1000BASE-LX",
+        },
+        "Ethernet28-32,Ethernet36,Ethernet40-44": {
+          "vendor_name": "Vendor/Inc",
+          "vendor_pn": "QSFP-100G-AOC-10M"
         }
       },
       "dut_name_2": {
-        "port_1": {
-          "vendor_name": "Vendor/Inc",
-          "vendor_pn": "QSFP-100G-AOC-10M"
+        "Ethernet0-96:4": {
```

Comment:
Should we use Python list slicing syntax instead?

Reply:
@az-pz I have addressed this now